ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -96,6 +100,18 @@ class MessagesHomeScreen extends React.Component<Props, State> {
 
   public componentDidMount() {
     this.props.refreshMessages();
+    this.navListener = this.props.navigation.addListener("didFocus", () => {
+      StatusBar.setBarStyle("dark-content");
```

Comment:
To avoid code duplication, could you create an _util_ function (also the relative file if it doesn't exists) to serve this goal?
something like this
```typescript
const setStatusBarColorAndBackround = (
  style: StatusBarStyle,
  color: string,
  animated: boolean = true
): void => {
  StatusBar.setBarStyle(style, animated);
  Platform.select({
    android: () => {
      StatusBar.setBackgroundColor(color, animated);
    }
  });
};
```